Packs/ApiModules/Scripts/HTTPFeedApiModule/HTTPFeedApiModule.py

```diff
@@ -214,6 +217,18 @@ def build_iterator(self, **kwargs):
                 results.append({url: result})
         return results
 
+    def custom_fields_creator(self, attributes: dict):
+        created_custom_fields = {}
+        for attribute in attributes.keys():
+            if attribute in self.custom_fields_mapping.keys():
+                created_custom_fields[self.custom_fields_mapping[attribute]] = attributes[attribute]
+
+        if len(created_custom_fields.keys()) > 0:
+            return None
+
+        else:
+            return created_custom_fields
```

Comment:
why do we need the elif? shouldn't it just be `return created_custom_fields` ?

Reply:
done